htmLawed.php

```diff
@@ -966,7 +966,7 @@ function hl_tag($t)
   if ($t == '>') {
     return '&gt;';
   }
-  if (!preg_match('`^<(/?)([a-zA-Z][^\s>]*)([^>]*?)\s?>$`m', $t, $m)) { // Get tag with element name and attributes
+  if (!preg_match('`^<(/?)([a-zA-Z][^\s>]*)([^>]*?)\s?(/?)>$`m', $t, $m)) { // Get tag with element name and attributes
```

Comment:
Maybe we should use named capture group, index 4 is just confusing.

```suggestion
  if (!preg_match('`^<(/?)([a-zA-Z][^\s>]*)([^>]*?)\s?(?P<selfclosing>/?)>$`m', $t, $m)) { // Get tag with element name and attributes
```

Reply:
Actually I'm afraid it would make it harder to keep up with upstream if we move to named capture groups. Are you suggesting this only for the self-closing tag or for all capture groups of this line as well?

Reply:
Yes only for our group.

Reply:
done 2295cf7be7e833214d15720c6b31c4cfed46c597